eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/ImpTest.hs

```diff
@@ -112,6 +117,23 @@ impLookupPlutusScript sh = do
     Just (ScriptTestContext plutus _) -> pure $ mkPlutusScript plutus
     Nothing -> pure Nothing
 
+impGetPlutusContexts ::
+  (ScriptsNeeded era ~ AlonzoScriptsNeeded era, EraUTxO era) =>
+  Tx era ->
+  ImpTestM
+    era
+    [(PlutusPurpose AsIxItem era, ScriptHash (EraCrypto era), ScriptTestContext)]
+impGetPlutusContexts tx = do
+  let
+    txBody = tx ^. bodyTxL
+  utxo <- getsNES $ nesEsL . esLStateL . lsUTxOStateL . utxosUtxoL
+  let
```

Comment:
I don't have any strong opinion on this, but these `let`s on a single line all by themselves look a bit odd.